src/main.rs

```diff
@@ -85,9 +87,18 @@ impl Options {
     }
 }
 
+fn pg_client(options: &Options) -> Result<Client, postgres::Error> {
+    let connector = TlsConnector::builder()
+        .danger_accept_invalid_certs(true)
+        .build()
+        .unwrap();
+    let connector = MakeTlsConnector::new(connector);
+    Client::connect(&options.database_url, connector)
+}
```

Comment:
You can avoid that `unwrap()` with:

```suggestion
fn pg_client(options: &Options) -> Result<Client, Box<dyn Error>> {
    let connector = TlsConnector::builder()
        .danger_accept_invalid_certs(true)
        .build()?;
    let connector = MakeTlsConnector::new(connector);
    Ok(Client::connect(&options.database_url, connector)?)
}
```